cmd/cmd_helpers_test.go

```diff
@@ -24,3 +20,18 @@ func executeCommandC(root *cobra.Command, args ...string) (string, error) {
 
 	return buf.String(), err
 }
+
+// mustPullPlugin pulls the gatewayd-plugin-cache plugin and returns the path to the archive.
```

Comment:
This is to prevent GitHub API rate limit to be reached by pulling the plugin once and reusing it (almost) everywhere.